pom.xml

```diff
@@ -59,7 +59,7 @@
         <derby.version>10.11.1.1</derby.version>
         <commons-io.version>2.4</commons-io.version>
         <kafka.connect.maven.plugin.version>0.11.1</kafka.connect.maven.plugin.version>
-        <sqlite-jdbc.version>3.8.11.2</sqlite-jdbc.version>
+        <sqlite-jdbc.version>3.25.2</sqlite-jdbc.version>
```

Comment:
Why upgrade this dependency within this PR? Why not just issue a separate, much smaller/simpler PR?

Reply:
@rhauch you are definitely right here.
I've created a different pull request to address this change.

https://github.com/confluentinc/kafka-connect-jdbc/pull/508